app/src/main/java/org/sopt/sample/data/remote/model/response/home/ResponseUserDto.kt

```diff
@@ -25,3 +26,13 @@ data class UserDTO(
     val avatar: String,
 )
 
+fun UserDto.toData(): User {
+    return User(
+        id = this.id,
+        name = this.firstName,
+        email = this.email,
+        profileUrl = this.avatar
+    )
+}
```

Comment:
```suggestion
fun UserDto.toData(): User = User(
        id = this.id,
        name = this.firstName,
        email = this.email,
        profileUrl = this.avatar
    )
```

Mapper 군용! 확장함수 사용 좋네요 :)